### DISCUSSION TITLE
Vision Pro support

### DISCUSSION THREAD

**Discussion by ceolinwill** [community/community#128958] [Mobile] [answered]
GitHub app already supports iPad. This means it should work with Apple Vision Pro as well. It's just a matter of enabling it.

I understand you might be concerned with support and user experience but, right now, the alternative is to use GitHub on Safari, which provides a much worse experience on touch interfaces than the mobile app.

Overall, the action areas on the web app are too small. This makes them difficult to be "clicked" on touch interfaces.

Some of the issues I found:

- Editing a README sometimes freezes.
- Scrolling a column on a project's board sometimes ends up dragging and dropping cards.
- When you actually want to drag and drop a specific card is nearly impossible. I always have to use the keyboard to do it.
- Clicking things, in general, is hard because the action area is too small.

All those issues could be minimized if GitHub allowed us to use the iPad app on Vision Pro.

**Comment by mattruggio** [comment, marked as answer]
hey @wceolin, thanks for the suggestion! I checked and we currently have an open internal iOS issue for adding Apple Vision Pro support. Unfortunately I cannot guarantee any specific timeline around it but it is something currently in the pipeline. Stay tuned!

**Reply by ceolinwill** [reply]
Awesome, thanks for the update. The iOS app is really great. Can’t wait to use it on Vision Pro.
